### PR TITLE
3 new endgame chems

### PR DESCRIPTION
**Pull request by Saeko-44**
**Changelog**

:cl:
- add: Abnormium, a new potent medicine made from grinding down abnormality cores. May cause cancer as a side effect.
- add: Philosopher's Juice, a catalyst for endgame chemistry made with unusual materials (more recipes involving this catalyst in the future)
- add: Holium, an impossibly hard to make chemical that will require the entire station to work in tandem to be made in the name of the cult of the hole

### CODE REVIEW

**Comment by Saeko-44** [comment]
For reference, the recipe for holium is made to be impossibly hard. Like. You'd need an entire station to work on this. Like cloning but as a stationwide effort.

https://discord.com/channels/716470896985899219/1263202625864597597/1315700001900462081

File: Resources/Prototypes/tags.yml [hunk 1]
```diff
@@ -563,6 +563,9 @@
 - type: Tag
   id: Flesh
 
+- type: Tag
```
Comment (hivehum) [inline Resources/Prototypes/tags.yml:566]: this should go in _impstation/tags.yml

**Comment by hivehum** [comment]
the cost seems fine for holium but im not personally sure how i feel about making hole entity something technically player accessible. im going to hold off on approving/merging and let admins decide

File: Resources/Prototypes/tags.yml (hunk 1 above)
Comment (Saeko-44) [reply Resources/Prototypes/tags.yml]: Ok I moved it

**Comment by formlessnameless** [comment]
okay. i am going to merge this for now. and then the first time it actually gets made in a round reevaluate. because it IS hard to make. but it's going to happen at some point.

**Review by formlessnameless — APPROVED** [review]

**Pull request merged**